### PR TITLE
Netappfiles update sdk version that include api version 2021 06 01

### PR DESCRIPTION
**Pull request by hildurhodd**
Update to sdk version 1.0.0-beta.5
Minimum changes made to support the new sdk version
- Use TokenCredential with AzureProfile instead of ServiceClientCredentials
- Use NetAppFilesManager instead of AzureNetAppFilesManagementClientImpl
- Remove Observables and CompletableFuture since it is not supported any more.
- Updated readme file to describe new way to authenticate

### CODE REVIEW

**Review by HalliSmiley — APPROVED** [review]

File: src/main/java/dualprotocol/sdk/sample/Creation.java [hunk 1]
```diff
@@ -20,18 +20,18 @@
...
         {
-            NetAppAccountInner anfAccount = anfClient.accounts().createOrUpdate(accountParams[0], accountParams[1], accountBody);
+            NetAppAccountInner anfAccount = anfClient.getAccounts().beginCreateOrUpdate(accountParams[0], accountParams[1], accountBody).getFinalResult();
             Utils.writeSuccessMessage("Account successfully created, resourceId: " + anfAccount.id());
 
-            return CompletableFuture.completedFuture(anfAccount);
+            return anfAccount;
         }
-        catch (CloudException e)
+        catch (Exception e)
```
Comment (paulomarquesc) [inline src/main/java/dualprotocol/sdk/sample/Creation.java:32]: >Exception [](http://example.com/codeflow?start=15&length=9)

CloudException got replaced by AzureError (at least in Python), and should not be replaced by the broader Exception, can you please apply the equivalent for Java? Please do the same on all samples.

File: README.md [hunk 2]
```diff
@@ -63,17 +63,22 @@ Make sure this file is stored at the root of the project.
...
+       $env:AZURE_CLIENT_ID = <clientId>
+       $env:AZURE_CLIENT_SECRET = <clientSecret>
+       $env:AZURE_TENANT_ID = <tenantId>
+       ```
+      Bash
+       ```bash
+       export AZURE_SUBSCRIPTION_ID=<subscriptionId>
+       export AZURE_CLIENT_ID=<clientId>
+       export AZURE_CLIENT_SECRET=<clientSecret>
+       export AZURE_TENANT_ID=<tenantId>
```
Comment (paulomarquesc) [inline README.md:80]: At least one sample must use the file authentication so customers can continue to see how it is done, let's leave this one using the way it was before this update.

**Review by paulomarquesc — CHANGES_REQUESTED** [review]
:clock1:

File: README.md (hunk 2 above)
Comment (hildurhodd) [reply README.md:80]: It looks like the newest versions of the SDK do not support ServiceClientCredentials.
I was not able to authenticate with it and therefor ended up authenticating as the SDK documentation recommend.

**Review by AdelGhabboun — APPROVED** [review]

**Review by paulomarquesc — APPROVED** [review]
:shipit:

**Pull request merged**